crux_http/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "crux_http"
 description = "HTTP capability for use with crux_core"
-version = "0.9.1"
+version = "0.9.2"
```

Comment:
The changelog says 0.9.2-rc.0 but this is just 0.9.2, I assume one of them is wrong

Reply:
Thanks! my bad